### PR TITLE
For Paddle2.1

### PR DESCRIPTION
**Pull request by SunAhong1993**
Please check the follow step before merging this pull request

- [x] Python code style verification
- [x] Review all the code diff by yourself
- [x] All models(TensorFLow/Caffe/ONNX/PyTorch) testing passed
- [ ] Details about your pull request, releated issues

If this PR add new model support, please update `model_zoo.md` and add model to out test model zoos(@wjj19950828)
- [ ] New Model Supported
- [x] No New Model Supported

## 修复问题
1. 由于Paddle2.1更新paddle.Tensor.dtype的用法，X2Paddle需要随之更新。

### CODE REVIEW

File: x2paddle/op_mapper/dygraph/pytorch2paddle/aten.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import copy
 import numpy as np
 from x2paddle.core.util import *
+from x2paddle.utils import *
```
Comment (jiangjiajun) [inline x2paddle/op_mapper/dygraph/pytorch2paddle/aten.py]: 代码中后面不要再使用这种用法，需要什么就import什么进来。之前有这样写法的，后面也要全改过来

File: x2paddle/project_convertor/pytorch/mapper.py [hunk 2]
```diff
@@ -169,11 +168,42 @@
     ["x2paddle.torch2paddle.init_process_group", None]
 }
 
-DTYPE_MAPPER = {
-    "torch.float32": [string("float32"), None],
-    "torch.long": [string("int64"), None],
-    "torch.bool": [string("bool"), None]
-}
+if is_larger_21:
```
Comment (jiangjiajun) [inline x2paddle/project_convertor/pytorch/mapper.py]: larger?

File: x2paddle/project_convertor/pytorch/torch2paddle/nn_init.py [hunk 3]
```diff
@@ -19,6 +19,7 @@
 from paddle.fluid.core import VarDesc
 from paddle.fluid.initializer import XavierInitializer, MSRAInitializer
 from paddle.fluid.data_feeder import check_variable_and_dtype
+from x2paddle.utils import *
```
Comment (jiangjiajun) [inline x2paddle/project_convertor/pytorch/torch2paddle/nn_init.py]: 同上

File: x2paddle/utils.py [hunk 4]
```diff
@@ -18,3 +18,34 @@ def string(param):
...
+
+import paddle
+version = paddle.__version__
+v0, v1, v2 = version.split('.')
+is_larger_21 = True
+if not ((v0 == '0' and v1 == '0' and v2 == '0') or
+        (int(v0) >= 2 and int(v1) >= 1)):
+    is_larger_21 = False
+
+if is_larger_21:
```
Comment (jiangjiajun) [inline x2paddle/utils.py]: 可以改成
from x2paddle.utils import paddle_dtypes
paddle_dtypes.float16 这样

File: x2paddle/op_mapper/dygraph/pytorch2paddle/aten.py (hunk 1 above)
Comment (SunAhong1993) [reply x2paddle/op_mapper/dygraph/pytorch2paddle/aten.py]: 已修改

File: x2paddle/project_convertor/pytorch/mapper.py (hunk 2 above)
Comment (SunAhong1993) [reply x2paddle/project_convertor/pytorch/mapper.py]: 已修改

File: x2paddle/project_convertor/pytorch/torch2paddle/nn_init.py (hunk 3 above)
Comment (SunAhong1993) [reply x2paddle/project_convertor/pytorch/torch2paddle/nn_init.py]: 已修改

File: x2paddle/utils.py (hunk 4 above)
Comment (SunAhong1993) [reply x2paddle/utils.py]: 已修改

**Pull request merged**